src/test/cli.build.test.ts

```diff
@@ -27,6 +27,17 @@ describe('Dev Containers CLI', function () {
 
 	describe('Command build', () => {
 
+		it('should fail to build with correct error message for local feature', async () => {
+			const testFolder = `${__dirname}/configs/image-with-local-feature`;
+			try {
+				await shellExec(`${cli} build --workspace-folder ${testFolder} --image-name demo:v1`);
+			} catch (error) {
+				const res = JSON.parse(error.stdout);
+				assert.equal(res.outcome, 'error');
+				assert.match(res.message, /prepend your Feature name with/);
```

Comment:
Nit: We can also assert for the entire error message.

Reply:
error message contains the dynamic feature name which is defined in the `devcontainer.json` file. Referred as `userFeatureId` where the error message is modified.
So used the match.